src/utils/fluid_sys.c

```diff
@@ -344,6 +344,19 @@ fluid_is_soundfont(const char *filename)
   return strncmp(id, "RIFF", 4) == 0;
 }
 
+/**
+ * Suspend the execution of the current thread for the specified amount of time.
+ * @param milliseconds to wait.
+ */
+void fluid_msleep(unsigned int msecs)
+{
+#ifdef _WIN32
+  Sleep(msecs);
```

Comment:
I dont see a necessity of having this special treatment for WIN32.

Reply:
You are right, it is not a strict necessity... since Windows Sleep() handles directly milliseconds, I introduced it and I avoided x1000, but if it looks unnecessary, we can fall down to g_usleep().

Reply:
Yes please. glib is doing all that platform dependent handling. We only move it to fluid_sys.c to ease getting rid of it for downstream if needed.